src/index.js

```diff
@@ -0,0 +1,121 @@
+import './style.css';
+import Ui, { LIST, VALUE } from './modules/ui.js';
+import Storage from './modules/storage.js';
+import Task from './modules/task.js';
+import DragDrop from './modules/dragDrop.js';
+
+// Display tasks from Local storage
+window.addEventListener('DOMContentLoaded', () => {
+  Ui.display();
+  DragDrop.addEventListeners();
+});
+
+// Add task
+const FORM = document.querySelector('form');
+FORM.addEventListener('submit', (e) => {
+  e.preventDefault();
+  const TASK_LIST = Storage.getEntry();
+  if (
+    VALUE.value !== ''
+    && !TASK_LIST.some((e) => e.description === VALUE.value)
+  ) {
```

Comment:
@aimalamiri, In this case, as is an input field, I used `VALUE.value !== ''` because an empty value doesn't return null, and for line 20, sure I will! thank you.

Reply:
You're correct, the operator `!==` will also check the value type.